.travis.yml

```diff
@@ -70,6 +70,7 @@ env:
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=1.7.4
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=2.0.2
     - TRAVIS_FLAVOR=elastic FLAVOR_VERSION=2.1.1
+    - TRAVIS_FLAVOR=snmp FLAVOR_VERSION=latest
```

Comment:
That's a good call, the image isn't using docker tags anyways, I'll remove the flavor_version.